lemarche/cms/snippets.py

```diff
@@ -28,4 +29,35 @@ def __str__(self):
         return self.name
 
 
-register_snippet(ArticleCategory)
+@register_snippet
+class Advert(models.Model):
+    title = models.CharField(max_length=255, verbose_name="Titre de l'encart")
+    image = models.ForeignKey(
+        "wagtailimages.Image", null=True, blank=True, on_delete=models.SET_NULL, related_name="+"
+    )
+
+    # call to action
+    url = models.URLField(null=True, blank=True)
```

Comment:
```suggestion
    cta_url = models.URLField(null=True, blank=True)
```
et je renommerais les champs suivants `cta_id`, `cta_text` & `cta_open_in_new_tab` :thinking: